packages/react-events/src/__tests__/Press-test.internal.js

```diff
@@ -1309,30 +1291,404 @@ describe('Event responder: Press', () => {
 
       ref.current.getBoundingClientRect = getBoundingClientRectMock;
       ref.current.dispatchEvent(
-        createPointerEvent('pointerdown', {
+        createEvent('pointerdown', {
+          pointerType: 'touch',
+        }),
+      );
+      ref.current.dispatchEvent(
+        createEvent('pointermove', {
+          ...coordinatesInside,
+          pointerType: 'touch',
+        }),
+      );
+      container.dispatchEvent(
+        createEvent('pointermove', {
+          ...coordinatesOutside,
+          pointerType: 'touch',
+        }),
+      );
+      container.dispatchEvent(
+        createEvent('pointermove', {
+          ...coordinatesInside,
+          pointerType: 'touch',
+        }),
+      );
+      container.dispatchEvent(
+        createEvent('pointerup', {
+          ...coordinatesInside,
+          pointerType: 'touch',
+        }),
+      );
+      jest.runAllTimers();
+
+      expect(events).toEqual([
+        'onPressStart',
+        'onPressChange',
+        'onPressMove',
+        'onPressEnd',
+        'onPressChange',
+        'onPressStart',
+        'onPressChange',
+        'onPressEnd',
+        'onPressChange',
+        'onPress',
+      ]);
+    });
+  });
+
+  describe('press with movement (touch events fallback)', () => {
```

Comment:
It's a little difficult to follow what happened in this part of the diff. Is this duplicating the entire pointer events test block? If so, I don't think we need to do that (and we don't for mouse events)

Reply:
It doesn't duplicate it all, a lot of it, but I changed a few tests too and remove a few there were mouse related.